zucchini-ui-frontend/src/failures/components/FailuresTable.jsx

```diff
@@ -0,0 +1,148 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Link} from 'react-router';
+import Table from 'react-bootstrap/lib/Table';
+import Label from 'react-bootstrap/lib/Label';
+import Modal from 'react-bootstrap/lib/Modal';
+import Status from '../../ui/components/Status';
+
+export default class FailuresTable extends React.Component {
+
+  render() {
+    const {failures} = this.props;
+    const rows = failures.failures.map(groupedFailures => {
```

Comment:
I think you should inject in your component only the `failures` array, not the full Redux store part.